curriculum/challenges/english/10-coding-interview-prep/rosetta-code/euler-method.english.md

```diff
@@ -95,6 +95,9 @@ tests:
 
 ```js
 function eulersMethod(x1, y1, x2, h) {
+  // x1 and y1 are initial conditions. y1 is initial temperature at x1 seconds
+  // h is step size
+  // time interval is from x1 seconds to x2 seconds
```

Comment:
To avoid having to translate these commented lines for the other languages, I think it is better to explain the parameters in the challenge description in terms of the order of the parameters.  For example, first parameter is..., second parameter is ..., etc.
```suggestion
```

Reply:
Thanks for the review. Your suggestion makes a lot more sense.